Xamarin.FacebookSlim/Xamarin.FacebookSlim.csproj

```diff
@@ -0,0 +1,73 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net6.0-ios;net6.0-maccatalyst</TargetFrameworks>
+    <IsBindingProject>true</IsBindingProject>
+    <NoBindingEmbedding>true</NoBindingEmbedding>
+    <RootNamespace>Xamarin.FacebookSlim</RootNamespace>
+    <SupportedOSPlatformVersion Condition="'$(TargetFramework)' == 'net6.0-ios'">14.0</SupportedOSPlatformVersion>
+    <SupportedOSPlatformVersion Condition="'$(TargetFramework)' == 'net6.0-maccatalyst'">14.0</SupportedOSPlatformVersion>
+    <BindingPlatform>$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)'))</BindingPlatform>
+    <AssemblyName>Xamarin.FacebookSlim</AssemblyName>
+  </PropertyGroup>
+  
+  <PropertyGroup>
+    <Copyright>Xamarin FacebookSlim 2022</Copyright>
+    <PackageProjectUrl>https://github.com/thisisthekap/Xamarin.FacebookSlim.iOS</PackageProjectUrl>
+    <PackageIcon></PackageIcon>
+    <Authors></Authors>
```

Comment:
fun.music IT GmbH, Hipolyto Obeso Huerta

Reply:
removed